src/SigFox.cpp

```diff
@@ -122,6 +122,12 @@ int SIGFOXClass::begin(SPIClass& spi, int reset, int poweron, int interrupt, int
 int SIGFOXClass::send(unsigned char mess[], int len, bool rx)
 {
   if (len == 0) return -1;
+
+  if (rx == false && len == 1 && mess[0] < 2) {
+    //we can use send_bit command
```

Comment:
:+1: @facchinm 

It's purely protocol-related .

On the backend/callback side, it's handled as an standard payload & exposed as an hexadecimal string to convert back to original data types.